server/games/custom_game.py

```diff
@@ -1,6 +1,8 @@
 import time
 
 from .game import Game, ValidityState
+from server.rating import RatingType
```

Comment:
For now this makes sense, but it won't anymore once you finish multiple rating support :P

Reply:
You want something else than an enum? Is it supposed to be populated from the db?